### PR TITLE
Exclude fields

### PR DESCRIPTION
**Pull request by joaojunior**
Create option to pass fields to not copy in history.

This new PR replaces the PR: #238

### CODE REVIEW

**Comment by joaojunior** [comment]
@gyermolenko @macro1 Hi, I send this PR to replaces the PR #238.
Here, I remove the mutable object in parameter of a function and I fix the error in tests with django 1.10

**Comment by macro1** [comment]
Looks good. It would be great to have this documented as well. Undocumented features can be frustrating both for contributors as well as users of the library. If you don't have time to add the documentation, we'll try to get it added when this is merged in the future.

**Comment by joaojunior** [comment]
@macro1 Very good! I will now write the docs for this.

**Comment by joaojunior** [comment]
@macro1 I already add the documentation about this.

**Comment by vhelke** [comment]
When is this released? We are really looking forward to it.  Cheers for the good work!

**Comment by leportella** [comment]
@joaojunior would you mind resolving the conflicts? Thank you!

File: docs/advanced.rst [hunk 1]
```diff
@@ -214,6 +214,32 @@ You can use the ``table_name`` parameter with both ``HistoricalRecords()`` or
 
     register(Question, table_name='polls_question_history')
 
+Choosing fields to not be stored
+--------------------------------
+
+It is possible to use the parameter ``excluded_fields`` to choice which fields
```
Comment (treyhunner) [inline docs/advanced.rst]: `choice` -> `choose` here

File: docs/advanced.rst [hunk 2]
```diff
@@ -214,6 +214,32 @@ You can use the ``table_name`` parameter with both ``HistoricalRecords()`` or
...
+
+For example, if you have the model:
+
+.. code-block:: python
+
+    class PollWithExcludeFields(models.Model):
+        question = models.CharField(max_length=200)
+        pub_date = models.DateTimeField('date published')
+
+And you don't want to stored the changes for the field ``pub_date``, it is necessary to update the model to:
```
Comment (treyhunner) [inline docs/advanced.rst]: ` to stored the changes` -> ` to store the changes`

**Pull request merged**